### ISSUE TITLE
Error in the HTTP2 framing layer issue before heroku-20

### ISSUE THREAD

**Issue by markcoxsmith** [curl/curl#6488]
We're using the ruby gem `ably-rest` version `1.1.4.rc` to utilise HTTP2

Last time we frequently encountered `Faraday::ConnectionFailed: Error in the HTTP2 framing layer` that made some requests fail.  

Would it be possible for you to help in investigating this error, could it be caused by the gem implementation?

We just got this error *Faraday::ConnectionFailed:* Error in the HTTP2 framing layer . Don't know what it does, and it looks like it resolved itself and retried.

**Comment by markcoxsmith** [comment]
The `typhoeus` library we swtiched to for HTTP/2 wraps libcurl which is where the error comes from.

The resulting fix was released in libcurl 7.65.0. Using Heroku, this means using Heroku-20 as this has version 7.68.0 whereas heroku-18 (for example) has 7.58.0

**TL;DR** To avoid this issue, please use libcrul 7.65.0 or above
